### DISCUSSION TITLE
NeMo Curator

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#20] [General]
Discuss things related to NeMo Curator here!

**Comment by itsnotmik** [comment]
NeMo Curator installation can be found here: https://github.com/NVIDIA/NeMo-Curator

Requires:
* Python 3.10
* CUDA 12x

**Comment by ACraig7** [comment]
NeMo Curator is [open source](https://github.com/NVIDIA/NeMo-Curator/blob/main/LICENSE).
